### DISCUSSION TITLE
No Bluetooth Light and Connection

### DISCUSSION THREAD

**Discussion by ROLL0829** [iRobotEducation/create3_docs#465] [Questions]
### How are you connecting to your Create 3?

Bluetooth (iRobot Coding)

### Computer(s) Model(s) and Operating System(s)

Windows 10 Enterprise

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

G.5.3

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

I am on page 7 of Create3-Welcome-Guide.pdf, step 4.
I am having trouble with the Bluetooth light in it because it does not turn on.
Referring to other discussions, I have confirmed the following.

　・Adapter Board is fully seated within the robot
　・Reboot on the Create3 Webserver
　・Yellow and Green indicators on Adapter Board are lit
　・Trying to connect Bluetooth in "iRobot Education" → impossible to do
　　(Consider the possibility that the indicator is just malfunctioning and Bluetooth is still functioning) 

I would like to know if there is any solution to this problem.

Sorry for the poor English by machine translation.

**Comment by shamlian** [comment]
Your English is great, there's nothing for which to apologize.

If there is no blue light on the adapter board with the toggle switch toward the Bluetooth symbol, but the yellow and green lights are illuminated, the BLE module most likely is not getting power, so there is potentially some kind of hardware fault. Photos of the adapter board could be helpful to help debug this problem (just to verify the switch position, and any potential obvious hardware problem). It would also be helpful to see if anything changes on the robot's log when the adapter board is inserted into the robot.

**Reply by ROLL0829** [reply]
Thank you for your response.

Attached is a photo of the board.
Attached is a log of when the board was inserted into the robot.
I have not fully analyzed the contents of the log, are there any hints in the log?

![image1](https://github.com/iRobotEducation/create3_docs/assets/152382582/de836e45-3d42-4a27-b550-948d1fb5cd35)
![image2](https://github.com/iRobotEducation/create3_docs/assets/152382582/84fce816-605e-4cc5-80a6-f89ecb9c35fe)
[messages[board_insert].txt](https://github.com/iRobotEducation/create3_docs/files/13511255/messages.board_insert.txt)

**Reply by shamlian** [reply]
Unfortunately, it seems to me that there is something wrong with the hardware of the adapter board. I'd recommend e-mailing education@irobot.com with a link to this thread.
